python/ray/tests/test_resource_demand_scheduler.py

```diff
@@ -421,6 +422,134 @@ def testScaleUpMinSanity(self):
         autoscaler.update()
         self.waitForNodes(2)
 
+    def testGetConcurrentResourceDemandToLaunch(self):
```

Comment:
Can you test this independently of StandardAutoscaler (remove the autoscaler)? Just test the get_nodes_to_launch function as some of the other unit tests in this file does.